libraries/eosiolib/contracts/eosio/action.hpp

```diff
@@ -132,18 +132,18 @@ namespace eosio {
     *  Verifies that @ref name exists in the set of provided auths on a action. Fails if not found.
     *
     *  @ingroup action
-    *  @param name - name of the account to be verified
+    *  @param n - name of the account to be verified
     */
    inline void require_auth( name n ) {
       internal_use_do_not_use::require_auth( n.value );
    }
 
    /**
-    *  Verifies that @ref name exists in the set of provided auths on a action.
+    *  This method is used to check if an account has any authorization. It verifies that name exists in the set of provided auths on an action. It works like @ref require_auth, but on @a fail returns bad result instead of breaking.
```

Comment:
... returns **false** result...

Reply:
returns `false` instead of expected value